sql/sys_vars.cc

```diff
@@ -6773,15 +6773,20 @@ static Sys_var_mybool Sys_binlog_encryption(
        READ_ONLY GLOBAL_VAR(encrypt_binlog), CMD_LINE(OPT_ARG),
        DEFAULT(FALSE));
 
-static const char *binlog_row_image_names[]= {"MINIMAL", "NOBLOB", "FULL", NullS};
+static const char *binlog_row_image_names[]=
+{
+  "MINIMAL", "NOBLOB", "FULL", "FULL_DIFFERENT", NullS
```

Comment:
Trying out another option of the name. How about
`FULL_NODUP` , where `NODUP` obviously :innocent: stands for ... what the FR actually wants, right? (That is only changed cols in `AI`)

Reply:
That is better.